packages/rum-core/test/benchmarks/001-performance-monitoring.bench.js

```diff
@@ -49,26 +37,24 @@ suite('PerformanceMonitoring', function() {
   configService.setConfig({ serviceName: 'benchmark-send-transactions' })
   configService.setConfig(agentConfig)
 
-  benchmark('createTransactionPayload', function() {
-    var tr = generateTransaction()
-    performanceMonitoring.createTransactionPayload(tr)
-  })
-
   function ResolvedPromise() {
     return Promise.resolve()
   }
+  const tr = generateTestTransaction(10)
+
+  benchmark('createTransactionPayload', function() {
+    performanceMonitoring.createTransactionPayload(tr)
+  })
 
   benchmark('sendTransactions-no-json', function() {
     apmServer._postJson = ResolvedPromise
-    var tr = generateTransaction()
```

Comment:
test should not get influenced by creating transaction and span object. I will create separate benchmarks for transaction and span creation time.

Reply:
It will happen in separate PR.

Reply:
Would you please add a todo item to the original issue?

Reply:
Sure